routers/userRouter.js

```diff
@@ -0,0 +1,7 @@
+const express = require("express");
+const router = express.Router();
+const { userController } = require("../controllers");
+
+router.post('/kakao/signin', userController.signInKakao)
```

Comment:
```suggestion
router.get('/kakao-signin', userController.signInKakao)
```
/kakao에서 한번 더 분기 될 필요가 없다면 위와 같이 end-point를 작성하는것이 좋을 것 같습니다. 이에 더해, 현재 service에 작성된 비즈니스 로직이 기존에 회원이었다면 login 처리를, 회원이 아니었다면 signup을 하고 있기 때문에 http method에 대한 정의가 혼란스러울 수 있으나 end-point를 `/auth/kakao-signin` 과 같이 `signin`으로 작성하셨다면 get method를 하는 것이 자연스럽습니다 :)

Reply:
get 메서드를 사용하면 프론트에서 토큰값을 보내줄 수가 없다고 합니다...?